sdks/java/core/src/main/java/org/apache/beam/sdk/options/GcsOptions.java

```diff
@@ -85,6 +88,31 @@
   void setGcsUploadBufferSizeBytes(Integer bytes);
 
   /**
+   * The class of the validator that should be created and used to validate paths.
+   * If pathValidator has not been set explicitly, an instance of this class will be
+   * constructed and used as the path validator.
+   */
+  @Description("The class of the validator that should be created and used to validate paths. "
+      + "If pathValidator has not been set explicitly, an instance of this class will be "
+      + "constructed and used as the path validator.")
+  @Default.Class(GcsPathValidator.class)
+  Class<? extends PathValidator> getPathValidatorClass();
```

Comment:
I believe names are merged across all options. So we should (in this or a future PR) rename this to something like `getGcsPathValidatorClass()`, or else this may collide with other options.